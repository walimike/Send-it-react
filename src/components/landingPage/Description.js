import React from "react";

const Description = () => {
  return (
    <div className="container">
        <div className="row">
            <div className="col s12 ">
                <div className="card-panel teal">
                    <span className="white-text">
                    SendIT is a courier service that helps users deliver parcels to different destinations.
                    SendIT provides courier quotes based on weight categories.
                    </span>
                </div>
            </div>
        </div>
    </div>
  );
};

export default Description;
