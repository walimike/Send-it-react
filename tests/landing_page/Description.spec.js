import React from "react";
import { shallow } from "enzyme";

import Description from "../../src/components/landingPage/Description";


describe("Description", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Description />);
    expect(wrapper).toMatchSnapshot();
  });
});