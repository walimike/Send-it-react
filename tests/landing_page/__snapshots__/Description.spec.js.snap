// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description should render without crashing 1`] = `ShallowWrapper {}`;
